Yehor/van_goux/index.html

```diff
@@ -4,164 +4,171 @@
+        </div>
+    </div>
 
-            <div class="features-blocks">
 
-                <div class="feature-block block1">
-                    <div class="feature-name">
-                        <img src="img/icons/shape1.svg" alt="shape1">
-                        <h1>Pixel perfect output</h1>
-                    </div>
-                    <p class="description">
-                        Crisp, pixel-perfect landing pages without bothering your developers
-                    </p>
-                </div>
+    <section class="features wrapper">
+        <header class="title">Features</header>
 
+        <div class="features-blocks">
 
-                <div class="feature-block block2">
-                    <div class="feature-name">
-                        <img src="img/icons/shape1.svg" alt="shape1">
-                        <h1>Pixel perfect output</h1>
-                    </div>
-                    <p class="description">
-                        Crisp, pixel-perfect landing pages without bothering your developers
-                    </p>
-                </div>
+            <div class="feature-block">
+                <div class="feature-name">
+                    <img src="img/icons/shape1.svg" alt="shape1">
+                    <h3>Pixel perfect output</h3>
+                </div>
+                <p class="description">
+                    Crisp, pixel-perfect landing pages without bothering your developers
+                </p>
+            </div>
+            <div class="feature-line"></div>
+            <div class="feature-block">
+                <div class="feature-name">
+                    <img src="img/icons/responsive2-website.svg" alt="shape1">
+                    <h3>Pixel perfect output</h3>
+                </div>
+                <p class="description">
+                    Crisp, pixel-perfect landing pages without bothering your developers
+                </p>
+            </div>
+            <div class="feature-line"></div>
+            <div class="feature-block">
+                <div class="feature-name">
+                    <img src="img/icons/icon2.svg" alt="shape1">
+                    <h3>Pixel perfect output</h3>
+                </div>
+                <p class="description">
+                    Crisp, pixel-perfect landing pages without bothering your developers
+                </p>
+            </div>
 
-                <div class="feature-block block3">
-                    <div class="feature-name">
-                        <img src="img/icons/shape1.svg" alt="shape1">
-                        <h1>Pixel perfect output</h1>
-                    </div>
-                    <p class="description">
-                        Crisp, pixel-perfect landing pages without bothering your developers
-                    </p>
-                </div>
+        </div>
 
+    </section>
+    <section class="forwho wrapper">
+        <header class="title hide-to-mob">Who is VanDoux for</header>

```

Comment:
You shoudn't use header as title !!! I've might be misspoken on our lection. I've meant that section should have heading tag (H1 - H6). Please check https://developer.mozilla.org/ru/docs/Web/HTML/Element/section